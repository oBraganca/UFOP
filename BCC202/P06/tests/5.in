1 6
1 61
1 24
1 87
1 100
4 89
4 87
3
5
4 76
2 90
2 5
2 91
3
4 35
4 11
5
-1