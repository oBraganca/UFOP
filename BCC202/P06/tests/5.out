Nao Pertence
Pertence
24 61 87 100 
Nao Pertence
Nao Pertence
Nao Pertence
61 87 100 
